src/generic/integral.cc

```diff
@@ -551,26 +551,26 @@ namespace oomph
     {0.08528361568265699, 0.014317320230681},
     {0.900399064086661, 0.08528361568265699},
     {0.014317320230681, 0.900399064086661}};
-  // Modified by DR 2017 to include factor 1/2
+  // Modified by AR 2023 to include factor 1/2
```

Comment:
@puneetmatharu is right (in his general comment on this pull request). We should briefly mention where this scheme is used (in the existing self-tests). Possibly not at all (yet -- until your FvK stuff is added) in which case, the relevant driver code may want to include a little stand-alone test that integrates the area of the domain -- this would be off by a factor of [dunno -- non-one] if the correction was wrong.

Reply:
Not used by anything (yet), should I put it in the comments?